docs/getting-started.md

```diff
@@ -1,68 +1,249 @@
 # Getting started
 
-The quickest way to get started with Prisma Client JS and/or Lift is by installing the Prisma CLI and running the interactive `init` command:
+This page explains how to get started with Prisma and your existing database (_brownfield_). 
+
+If you don't have a database to try Prisma or just want to explore everything from scratch, check out the ready-to-run [example projects](https://github.com/prisma/prisma-examples/) (_greenfield_) with use cases for REST, GraphQL, gRPC and more.
+
+## TLDR
+
+Follow these steps to use Prisma with your existing database. Note that these steps assume that you have an existing Node.js or TypeScript project (in case you don't, follow the [extended guide](#extended-guide) below):
+
+1. Install `prisma2` as a development dependency: `npm install prisma2 --save-dev`
+1. Run `npx prisma2 init .` to create an empty [Prisma schema](./prisma-schema-file.md)
```

Comment:
`init` doesn't take any params currently, so the dot should not be necessary

Reply:
For me it does take a folder name, e.g. `prisma2 init hello-world` creates `hello-world/prisma/schema.prisma`.